common/scripts/delete-csv.sh

```diff
@@ -15,25 +15,25 @@
 # limitations under the License.
 #
 set -e
-QUAY_NAMESPACE=${QUAY_NAMESPACE:-opencloudio}
```

Comment:
This file likely isn't really used anymore, it was previously used to delete "applications" from quay.io before we moved to the catalog install method early in the transition to operators.  It doesn't really matter what you do with it since we're not using it anymore, but it doesn't actually need to be updated from quay to ICR (and in fact doesn't make sense to since there isn't a concept of applications on ICR in this manner).